lib/src/ast/element.dart

```diff
@@ -97,6 +101,19 @@ abstract class ElementAst implements StandaloneTemplateAst {
   /// Reference assignments.
   List<ReferenceAst> get references;
 
+  /// Bananas assignments.
+  List<BananaAst> get bananas;
+
+  ///Star assignments.
+  List<StarAst> get stars;
+
+  set bananas(List<BananaAst> l);
```

Comment:
This is primarily because desugaring process removes BananaAst from bananas and similar for stars by just setting a const []. If this isn't preferable, I can just simply copy and set cont [] for those two lists.